### PR TITLE
WINDUPRULE-623: springboot jmx to quarkus rule

### PR DESCRIPTION
**Pull request by m-brophy**
https://issues.redhat.com/browse/WINDUPRULE-623

### CODE REVIEW

File: rules-reviewed/quarkus/springboot/tests/data/springboot-jmx/SpringbootAnnotationConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    @ManagedAttribute(description="Item", currencyTimeLimit=15)
+    public int getItem() {
+        return item;
+    }
+
+    public void setItem(int item) {
+        this.item = item;
+    }
+
+}
```
Comment (jonathanvila) [inline rules-reviewed/quarkus/springboot/tests/data/springboot-jmx/SpringbootAnnotationConfiguration.java:20]: Better to add newline at end of file : https://thoughtbot.com/blog/no-newline-at-end-of-file

File: rules-reviewed/quarkus/springboot/tests/data/springboot-jmx/springboot-jmx-configuration.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            </map>
+        </property>
+    </bean>
+
+    <bean id="testBean" class="org.springframework.jmx.JmxTestBean">
+        <property name="name" value="TEST"/>
+        <property name="age" value="100"/>
+    </bean>
+
+</beans>
```
Comment (jonathanvila) [inline rules-reviewed/quarkus/springboot/tests/data/springboot-jmx/springboot-jmx-configuration.xml:20]: Better to add newline at end of file : https://thoughtbot.com/blog/no-newline-at-end-of-file

File: rules-reviewed/quarkus/springboot/tests/springboot-jmx-to-quarkus.windup.test.xml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                        <hint-exists message="Spring JMX annotation configuration detected:"/>
+                    </not>
+                </when>
+                <perform>
+                    <fail message="[springboot-jmx-to-quarkus-extension] SpringBoot JMX is not supported by Quarkus was not found!" />
+                </perform>
+            </rule>
+        </rules>
+    </ruleset>
+</ruletest>
```
Comment (jonathanvila) [inline rules-reviewed/quarkus/springboot/tests/springboot-jmx-to-quarkus.windup.test.xml:33]: Better to add newline at end of file : https://thoughtbot.com/blog/no-newline-at-end-of-file

**Review by jonathanvila — APPROVED** [review]

**Comment by jonathanvila** [comment]
It's OK for me not to add links to documentation as there is no official statement on not supporting JMX

File: rules-reviewed/quarkus/springboot/springboot-jmx-to-quarkus.windup.xml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+                    <message>
+                        Spring JMX annotation configuration detected:
+
+                        Spring JMX is not supported by Quarkus with GraalVM on a Native Image
+                    </message>
+                </hint>
+            </perform>
+        </rule>
+    </rules>
+</ruleset>
```
Comment (jonathanvila) [inline rules-reviewed/quarkus/springboot/springboot-jmx-to-quarkus.windup.xml:50]: Better to add newline at end of file : https://thoughtbot.com/blog/no-newline-at-end-of-file

**Comment by m-brophy** [comment]
newlines added to file ends

**Comment by jonathanvila** [comment]
LGTM :+1:

**Pull request merged**